website/blog/2024-08-06-langgraph-model-from-code/index.md

```diff
@@ -67,13 +66,12 @@ def validate_langgraph_environment_variables():
     """Ensure that required secrets and project environment variables are present."""
 
     # Validate enviornment variable secrets are present
-    required_secrets = ["OPENAI_API_KEY", "LANGSMITH_API_KEY"]
+    required_secrets = ["OPENAI_API_KEY"]
 
     if missing_keys := [key for key in required_secrets if not os.environ.get(key)]:
         raise ValueError(f"The following keys are missing: {missing_keys}")
 
     # Add project environent variables if not present
-    os.environ["LANCHAIN_TRACING_V2"] = os.environ.get("LANGCHAIN_TRACING_V2", "true")
     os.environ["LANGCHAIN_PROJECT"] = os.environ.get(
         "LANGCHAIN_TRACING_V2", "LangGraph MLflow Tutorial"
     )
```

Comment:
Can we avoid setting `LANGCHAIN_PROJECT` as well? This environment variable is for enabling LangSmith tracing.